### ISSUE TITLE
Do your magic!

### ISSUE THREAD

**Issue by DrMattG** [SORTEE/Checklists#1]
@EIvimeyCook 
 https://github.com/SORTEE/Checklists/tree/main/inst/shiny-examples/EcochecklistR/EcochecklistR - add the shiny app here and I will do all the packaging etc.

**Comment by EIvimeyCook** [comment]
Sweet - so let's just recap. We want an app where you can select various key terms and it returns an appropriate checklist for you? :) @DrMattG

**Comment by DrMattG** [comment]
Yeah I think so! We can use the table as the basis for the app so it updates when we update the list. I’ve made it an r package structure so we can have GitHub pages and GitHub actions to automate updates to shiny

**Comment by EIvimeyCook** [comment]
wait - was i meant to do this? hahah
